content/4_Gold/TopoSort.mdx

```diff
@@ -644,7 +652,7 @@ public class Main {
 
 		PrintWriter pw = new PrintWriter(System.out);
 
-		ArrayDeque<Integer> answer = new ArrayDeque<Integer>(); //Acts as a Stack
+		ArrayDeque<Integer> answer = new ArrayDeque<>(); //Acts as a Stack
 		int temp = N-1;
```

Comment:
```suggestion
		int temp = N - 1;
```
can you add spaces in all places that need them (can't add suggestions but lines 559, 561, 668, 670 and there's probably more that i didn't find)